docs/pages/foundations/rtl_guidelines/icons.js

```diff
@@ -0,0 +1,653 @@
+// @flow strict
+import React, { type Node } from 'react';
+// eslint-disable-next-line no-unused-vars
+import { Box, Callout, Flex, Heading, Icon, Image, Mask, Table, Text } from 'gestalt';
+import { DOCS_COPY_MAX_WIDTH_PX } from '../../../docs-components/consts.js';
+import MainSection from '../../../docs-components/MainSection.js';
+import Markdown from '../../../docs-components/Markdown.js';
+import Page from '../../../docs-components/Page.js';
+import PageHeader from '../../../docs-components/PageHeader.js';
+
+export default function FormsLayoutOverview(): Node {
+  return (
+    <Page title="Icons">
+      <PageHeader
+        name="Icons"
+        type="guidelines"
+        description={`
+In addition to layout and alignment, it is also important to consider icons. When using icons for RTL web and native app development, consider factors such as icon direction and context. For example, you should use CSS transforms or SVG attributes to flip icons depending on the text direction. Additionally, it is important to make sure the icons are appropriate and relevant for the context and culture of your target audience.
+
+Instead of going out to find your own icons, please leverage existing icons and components as they are consistent with our design language and can often handle RTL needs out-of-the-box.
+        `}
+      />
+      <MainSection
+        name="Leverage existing components and icons"
+        description="Our Icon component handles CSS transforms and SVG attributes to mirror icons for RTL. Additionally, we have a robust icon library that should be used in all Pinterest products. If you find issues with icons, or need to adapt an icon, [submit a request](https://gestalt.pinterest.systems/team_support/component_request#Filling-a-request-form)"
+      >
+        <MainSection
+          name="Icon types that need to be mirrored"
```

Comment:
nit: Can this H2 title be shorter.

Mirrored Icons and Non-Mirrored Icons (sometimes the "don't" is easy to miss)

<img width="295" alt="image" src="https://github.com/pinterest/gestalt/assets/5509813/0bdadf73-b88f-4245-9825-b572d8704df3">

Reply:
Perhaps, but stakeholders have already approved, and I'd rather wait for a v2 for additional changes. It's a bit long, but I think it's better to be very clear.